src/libraries/System.Net.Quic/src/System.Net.Quic.csproj

```diff
@@ -12,7 +12,9 @@
     <DefineConstants Condition="'$(TargetPlatformIdentifier)' == 'windows'">$(DefineConstants);TARGET_WINDOWS</DefineConstants>
     <GeneratePlatformNotSupportedAssemblyMessage Condition="'$(TargetPlatformIdentifier)' == ''">SR.SystemNetQuic_PlatformNotSupported</GeneratePlatformNotSupportedAssemblyMessage>
     <ApiExclusionListPath Condition="'$(TargetPlatformIdentifier)' == ''">ExcludeApiList.PNSE.txt</ApiExclusionListPath>
-    <UseQuicTransportPackage  Condition="'$(UseQuicTransportPackage)' == ''">false</UseQuicTransportPackage>
+    <!-- This controls if we consume official binaries from MsQuic or if we use binaries published from dotnet/msquic repo.
```

Comment:
```suggestion
    <!-- This controls if we consume official MsQuic binaries for Windows from NuGet (false) or if we use binaries published from dotnet/msquic repo (true).
```

Reply:
I think the transport is also NuGet - just different feed. I'm happy to make updates that make sense to us.

Reply:
From NuGet.org? :smile:

Reply:
the comment did not say NuGet.org :)
and I'm not sure it matters.